Workbooks/CosmosDb/Resource Insights/Resource Insights.workbook

```diff
@@ -69,17 +69,61 @@
             "value": {
               "durationMs": 86400000
             }
+          },
+          {
+            "id": "82f8fb72-8e01-4516-8115-b62082df1e44",
+            "version": "KqlParameterItem/1.0",
+            "name": "Database",
+            "type": 2,
+            "description": "When the value is \"unset\" or not selected, the data being displayed is at the account level.",
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Resources}/sqlDatabases\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2020-04-01\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value.*.properties.resource.id\",\"columns\":[]}}]}",
+            "value": null,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "defaultValue": "value::1",
```

Comment:
we should the remove the `defaultValue` part, because sometimes it 'chooses' a value in the background and it doesn't seem to make it appear on the dropdown (might be a bug on our end)